packages/serverless-cdk-plugin/package.json

```diff
@@ -74,5 +78,12 @@
         ]
       }
     }
+  },
+  "typesVersions": {
+    "*": {
+      "helpers": [
+        "./dist/types/helpers.d.ts"
+      ]
+    }
```

Comment:
Is this necessary?

Reply:
This part is the continuation of our atypical method(: